### PR TITLE
Fix unsafe array access 

### LINKED ISSUE #13098: Undefined offset at admin_audit/lib/Actions/Console.php
### Steps to reproduce
1. sudo -u www-data php -f ./occ
2. check last line in log

### Expected behaviour
Nothing should be logged (no error)

### Actual behaviour
```
{"reqId":"W7IFTuV5jg7KxHu2eZLD","level":3,"time":"2018-12-16T17:59:19+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"Undefined offset: 1 at \/var\/www\/nextcloud\/apps\/admin_audit\/lib\/Actions\/Console.php#33","userAgent":"--","version":"15.0.0.10"}
```

### Server configuration
Debian stretch, Apache, PostgreSQL 11, PHP 7.2, Nextcloud 15 (fresh install)

$arguments[1] is accessed, which causes an exception if no arguments have been issued.

For an almost no-op module written in 2017 and even made "strict" in 2018, this is unacceptable from the code quality point of view.

### PR DESCRIPTION
**Pull request by kesselb**
Close #13098 

I see this on my instance but cannot reproduce :see_no_evil:

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: apps/admin_audit/lib/Actions/Console.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Console extends Action {
 	 * @param $arguments
 	 */
 	public function runCommand(array $arguments) {
-		if ($arguments[1] === '_completion') {
+		if (isset($arguments[1]) && $arguments[1] === '_completion') {
```
Comment (nickvergessen) [inline apps/admin_audit/lib/Actions/Console.php]:
```suggestion
		if (!isset($arguments[1]) || $arguments[1] === '_completion') {
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
Can easily reproduce, just run `sudo -u www-data occ` with nothing else :see_no_evil: 
I think we shouldn't log this, so my code suggestion.

**Comment by kesselb** [comment]
> Can easily reproduce, just run `sudo -u www-data occ` with nothing else see_no_evil
> I think we shouldn't log this, so my code suggestion.

:+1: can reproduce now. admin_audit was broken somehow. your suggestion make sense. no need to log occ without arguments.

**Review by nickvergessen — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**